test/tests/actioncomponents/gold/saved_mesh_component_out.e

```diff
@@ -0,0 +1 @@
+two_same_components_out.e
```

Comment:
So this is some feature of exodus files - that you can just list the name of another file?

Reply:
this is how git shows the symlinks